tabfilter.py

```diff
@@ -16,25 +16,35 @@ class TabFilterCommand(sublime_plugin.WindowCommand):
 
 	def run(self):
 		"""Shows a quick panel to filter and select tabs from the active window"""
-		
+
 		tabs = []
 		self.window = sublime.active_window()
 		self.views = []
 		self.prefix = ""
 		self.settings = sublime.load_settings("tabfilter.sublime-settings")
 
+		idx = 0
 		for view in self.window.views():
 			self.views.append(view)
+			if self.window.active_sheet().view().id() == view.id():
+				""" save index for later usage"""
+				self.current_tab_idx = idx
 			tabs.append(self.make_tab(view))
+			idx = idx + 1
 
 		self.prefix = len(os.path.commonprefix([entity.name for entity in tabs if entity.is_file]))
 
 		show_captions = self.settings.get("show_captions", True)
 		include_path = self.settings.get("include_path", False)
+		preview_tab = self.settings.get("preview_tab", False)
 
 		tabs = [entity.get_details(self.prefix, include_path, show_captions) for entity in tabs]
 
-		self.window.show_quick_panel(tabs, self._on_done)
+		if preview_tab is True and self.window.num_groups() == 1:
```

Comment:
I'm going to admit, this is definitely me being incredibly pedantic, but I go out of my way to try and avoid else blocks in code if possible - generally I'm of the mind that it's the default outcome of the `if` not passing, so we shouldn't need to force it into an enclosed block unless there's something amiss.  It's not a hill I'd die on, but it is something I try to code around, 'cause in bigger projects or bigger pieces of code, if/else hell can often be an indicator that something else in the design is shaky.

In this scenario, we can avoid the else by defaulting and overriding the optional parameters we're passing through, so it becomes something more like:

```python
on_highlight_cb = None
selected_index = None

if preview_tab is True and self.window.num_groups() == 1:
    # This doesn't work with a multi group layout
    # but otherwise, replace our defaulted arguments.
    on_highlight_cb = self._on_highlighted
    selected_index = self.current_tab_idx

self.window.show_quick_panel(tabs, self._on_done, on_highlight=on_highlight_cb, selected_index=selected_index)
```